8.8/vips.modules

```diff
@@ -467,76 +421,42 @@
     </after>
   </cmake>
 
-    -->
-
-  <autotools id="poppler" 
-    autogen-sh="configure"
-    autogenargs="--with-font-configuration=win32 --disable-xpdf-headers --disable-splash-output --enable-poppler-glib --disable-gtk-test --disable-utils --disable-poppler-cpp" 
-    makeargs="POPPLER_GLIB_DISABLE_DEPRECATED="
-    >
-    <pkg-config>poppler-glib.pc</pkg-config>
-    <branch repo="poppler.freedesktop.org"
-            module="poppler-0.46.0.tar.xz"
-            version="0.46.0"
-      />
-    <dependencies>
-      <dep package="cairo"/>
-      <dep package="openjpeg"/>
-      <dep package="glib"/>
-    </dependencies>
-    <after>
-      <dep package="gdk-pixbuf"/>
-    </after>
-  </autotools>
-
-  <autotools id="fftw3" 
+  <autotools id="fftw3"
     autogen-sh="configure"
-    autogenargs="--disable-static --enable-shared --disable-threads"
-    >
+    autogenargs="--disable-static --enable-shared --disable-threads">
     <branch
       repo="fftw"
-      module="fftw-3.3.7.tar.gz"
-      version="3.3.7"
-    />
+      module="fftw-3.3.8.tar.gz"
+      version="3.3.8"/>
     <dependencies>
       <dep package="gettext"/>
     </dependencies>
   </autotools>
 
-  <autotools id="lcms" 
-    autogen-sh="configure"
-    >
+  <autotools id="lcms"
```

Comment:
Not related to the Windows builds but perhaps we should look to `lcms2mt`, see:
https://github.com/ArtifexSoftware/thirdparty-lcms2/blob/05db680de2d7178be21450bc0cc909460c7d4aeb/doc/WhyThisFork.txt

According to that document, LCMS can be used in multi-threaded environment.